Lib/test/test_curses.py

```diff
@@ -1107,6 +1107,39 @@ def test_issue6243(self):
         curses.ungetch(1025)
         self.stdscr.getkey()
 
+    def test_in_wch(self):
+        stdscr = self.stdscr
+
+        if not hasattr(stdscr, 'in_wch'):
+            raise unittest.SkipTest('requires curses.window.in_wch')
```

Comment:
Do you need this kind of check or can you use `@requires_curses_window_meth('in_wch')`?